src/components/MessageList.js

```diff
@@ -1,16 +1,29 @@
 import React from "react";
+import MessageBubble from "./MessageBubble";
+import MyMessageBubble from "./MyMessageBubble";
 
 export default class MessageList extends React.Component {
   render() {
     return (
       <div className="message-list">
+        <div className="date">
+          <div className="date-text"> Today </div>
+        </div>
         {this.props.messages.map((message, index) => {
-          return (
-            <div key={index} className="message">
-              <div className="message-username">{message.sender}</div>
-              <div className="message-text">{message.text}</div>
-            </div>
-          );
+          if (message.sender === "First") {
```

Comment:
add state some where and get state to compare
```suggestion
          if (message.sender === this.state.username) {
```